docs/registry.md

```diff
@@ -0,0 +1,58 @@
+# Configure Image Registry for CRI-containerd
+This document describes the methods how to configure the image registry for CRI-containerd.
+
+In CRI-containerd, `docker.io` is the default image registry. You can also set  up other image registries similar to docker. CRI-containerd provides two ways to set up the image registry.
+
+## Configure Image Registry with `--registry` Option
+
+Start CRI-containerd with `--registry` option like:
+```bash
+$ cri-containerd \
+    --registry=test.secure-registry.io=https://HostIP1:Port1 \
+    --registry=test.insecure-registry.io=http://HostIP2:Port2
+```
+
+If you want configure a insecure registry, you need set the endpoint with `http://` prefix. If you want configure a secure registry, you need set the endpoint with `https://` prefix.
+
+This method is only suitable for standalone mode.
+
+
+## Configure Image Registry with Config File in Standalone Mode
+In standalone mode, CRI-containerd also specifies registris using the configuration file located in `/etc/cri-containerd/config.toml`. You can set registries in the configuration file as follows:
+```toml
+#... ...
+[registry]
+  [registry.mirrors]
+    [registry.mirrors."docker.io"]
+      endpoint = ["https://registry-1.docker.io"]
+    [registry.mirrors."test.secure-registry.io"]
+      endpoint = ["https://HostIP1:Port1"]
+    [registry.mirrors."test.insecure-registry.io"]
+      endpoint = ["http://HostIP2:Port2"]
+```
+
+In the same way, the default configuration also can be generated by `cri-containerd default-config > /etc/cri-containerd/config.toml`. The endpoint is a list that can contain multiple image registry URLs splited by commas.
```

Comment:
Move `The endpoint is a list that can contain multiple image registry URLs splited by commas.` up to a separate paragraph.